packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_sidebar.dart

```diff
@@ -26,17 +28,17 @@ class _PropertiesList extends StatelessWidget {
   const _PropertiesList();
 
   static const itemPadding = densePadding;
-  static const itemDivider = Divider(thickness: 1.0);
 
   @override
   Widget build(BuildContext context) {
+    // TODO(https://github.com/flutter/devtools/issues/8546) Switch to scrollable
+    // ListView when this has been moved into its own panel.
     return Column(
       children: [
-        itemDivider,
-        for (final property in _properties) ...[
-          _EditablePropertyItem(property: property),
-          itemDivider,
-        ],
+        for (final property in _properties)
+          ...<Widget>[
+            _EditablePropertyItem(property: property),
+          ].joinWith(const PaddedDivider.noPadding()),
```

Comment:
this looks like it is calling joinWith on a list of size 1 for each iteration in the loop. This should give us what we want:
```
children: _properties.map((p) => _EditablePropertyIdem(property: p)).joinWith(const PaddedDivider.noPadding())
```

(a `toList()` call may be needed in there if we get analysis errors)

Reply:
Oops, thanks for catching that! Done